libs/tdb/files/tdb.cache.txt

```diff
@@ -0,0 +1,36 @@
+Checking simple C program: OK
```

Comment:
@pprindeville I concern about this either, so I tested the tdb.cache.txt file under gcc 5.x, gcc-6.x, gcc-7.x with LEDE trunk, it works fine. Furthermore, once these values are no longer correct, turn them to 'No' can make it. :-)